diff --git a/src/fabric/SearchBarNativeComponent.ts b/src/fabric/SearchBarNativeComponent.ts
index df774b955fffe8b6bb76883da3b12a2db746b9f7..d8d834d982e10b658496afd825c0eca45ca74a9d 100644
--- a/src/fabric/SearchBarNativeComponent.ts
+++ b/src/fabric/SearchBarNativeComponent.ts
@@ -9,7 +9,7 @@ import type {
 } from 'react-native';
 
 // eslint-disable-next-line @typescript-eslint/ban-types
-export type SearchBarEvent = Readonly<{}>;
+export type SearchBarEvent = Readonly<{ placeholder?: string }>;
 
 export type SearchButtonPressedEvent = Readonly<{
   text?: string;
